ffc-pay-mi-reporting/mi-report.js

```diff
@@ -55,6 +60,7 @@ const parseEventData = (eventData) => {
     totalAmount: paymentData.value,
     batchId: sequence,
     batchCreatorId: paymentData.sourceSystem,
+    exportDate: batchExportDate,
```

Comment:
Suggest we just keep this as `batchExportDate` to be clear what it is.  Also matches the name on their existing MI report